Internal Server Error: /payment/1/withdraw/request/
"POST /payment/1/withdraw/request/ HTTP/1.1" 500 66
Internal Server Error: /payment/1/withdraw/request/
"POST /payment/1/withdraw/request/ HTTP/1.1" 500 66
Internal Server Error: /payment/1/withdraw/request/
"POST /payment/1/withdraw/request/ HTTP/1.1" 500 66
Internal Server Error: /auth/reset/password
Traceback (most recent call last):
  File "/home/quarjo/pollproject/env/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/quarjo/pollproject/env/lib/python3.12/site-packages/django/utils/deprecation.py", line 131, in __call__
    response = self.process_response(request, response)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/quarjo/pollproject/env/lib/python3.12/site-packages/django/middleware/common.py", line 108, in process_response
    return self.response_redirect_class(self.get_full_path_with_slash(request))
                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/quarjo/pollproject/env/lib/python3.12/site-packages/django/middleware/common.py", line 87, in get_full_path_with_slash
    raise RuntimeError(
RuntimeError: You called this URL via POST, but the URL doesn't end in a slash and you have APPEND_SLASH set. Django can't redirect to the slash URL while maintaining POST data. Change your form to point to localhost:8000/auth/reset/password/ (note the trailing slash), or set APPEND_SLASH=False in your Django settings.
"POST /auth/reset/password HTTP/1.1" 500 73182
Internal Server Error: /auth/reset/password
Traceback (most recent call last):
  File "/home/quarjo/pollproject/env/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/quarjo/pollproject/env/lib/python3.12/site-packages/django/utils/deprecation.py", line 131, in __call__
    response = self.process_response(request, response)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/quarjo/pollproject/env/lib/python3.12/site-packages/django/middleware/common.py", line 108, in process_response
    return self.response_redirect_class(self.get_full_path_with_slash(request))
                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/quarjo/pollproject/env/lib/python3.12/site-packages/django/middleware/common.py", line 87, in get_full_path_with_slash
    raise RuntimeError(
RuntimeError: You called this URL via POST, but the URL doesn't end in a slash and you have APPEND_SLASH set. Django can't redirect to the slash URL while maintaining POST data. Change your form to point to localhost:8000/auth/reset/password/ (note the trailing slash), or set APPEND_SLASH=False in your Django settings.
"POST /auth/reset/password HTTP/1.1" 500 73182
